src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Basic.cs

```diff
@@ -101,7 +100,9 @@ public sealed class Delete : InstructionBase
 
         public override void Execute(ActionContext context)
         {
-            throw new NotImplementedException();
+            var value = context.Pop();
```

Comment:
This is not correct according to SWF specification:
https://www.adobe.com/content/dam/acom/en/devnet/pdf/swf-file-format-spec.pdf

I think you should first **Pop** the name, then the object and then deletect the variable from that object

Reply:
i removed it for now